npm/angular/package.json

```diff
@@ -59,6 +58,18 @@
   "publishConfig": {
     "access": "public"
   },
+  "nx": {
+    "targets": {
+      "build": {
+        "inputs": [
+          "{workspaceRoot}/scripts"
+        ],
+        "outputs": [
+          "{workspaceRoot}/cli/angular"
```

Comment:
Because we copy these assets to the `/cli` directory we need to specify them here.